ajax/libs/bulmaswatch/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bulmaswatch",
+  "description": "Themes for Bulma",
+  "author": "Jenil Gogari <i@jgog.in>",
+  "homepage": "https://jenil.github.io/bulmaswatch",
+  "filename": "bulmaswatch.min.css",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jenil/bulmaswatch.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "css",
+    "sass",
+    "flexbox",
+    "responsive",
+    "framework",
+    "themes"
+  ],
+  "autoupdate": 
+  {
```

Comment:
The position of this '{' is a little bit not in our format, but the review bot didn't catch it , so I think its fine.